sources/packages/web/src/components/aest/students/EditCASSupplierInfoModal.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <v-form ref="addCASSupplierForm">
+    <modal-dialog-base
+      title="Update supplier/Site information"
```

Comment:
I can't think of a single reason that would justify the "Site" being in upper case.
Can we have it adjusted as `Update supplier/site information`?
If not, we can wait for some content review in this UI.